content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -32,37 +32,46 @@ different approach.
 
 ## The Kubernetes network model
 
-Every `Pod` gets its own IP address. This means you do not need to explicitly
-create links between `Pods` and you almost never need to deal with mapping
-container ports to host ports.  This creates a clean, backwards-compatible
-model where `Pods` can be treated much like VMs or physical hosts from the
-perspectives of port allocation, naming, service discovery, load balancing,
-application configuration, and migration.
+Every `Pod` gets its own IP address, maximum one per IP family. This means
+you do not need to explicitly create links between `Pods` and you almost
+never need to deal with mapping container ports to host ports.  This creates
+a clean, backwards-compatible model where `Pods` can be treated much like
+VMs or physical hosts from the perspectives of port allocation, naming,
+service discovery, load balancing, application configuration, and migration.
+
+Kubernetes IP addresses exist at the `Pod` scope, in the `status.PodIPs` field
+- containers within a `Pod` share their network namespaces - including their IP
+address. This means that containers within a `Pod` can all reach each other's
+ports on `localhost`. This also means that containers within a `Pod` must
+coordinate port usage, but this is no different from processes in a VM. This is
+called the _IP-per-pod_ model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (on platforms that support it). Even if a host has
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, that is, the `status.PodIPs` field will only ever contain one
```

Comment:
Can we avoid wording like "will only ever" - that's a promise I don't think we should make.  Maybe "the `status.PodIPs` field only contains one IP per address family (for now)"